pr_agent/settings/pr_reviewer_prompts.toml

```diff
@@ -20,6 +20,12 @@ You must use the following JSON schema to format your answer:
       "type": "string",
       "enum": ["Bug fix", "Tests", "Bug fix with tests", "Refactoring", "Enhancement", "Documentation", "Other"]
     },
+{%- if require_score %}
+    "Score": {
```

Comment:
The 'Score' field in the JSON schema should be optional, as not all PRs may require a score. Consider adding a 'required' attribute to the 'Score' field and set it to false. [medium]